aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -112,6 +113,18 @@ Additional HTTP.sys configuration is handled through [registry settings](https:/
 
 For more information about HTTP.sys options, see <xref:Microsoft.AspNetCore.Server.HttpSys.HttpSysOptions>.
 
+### Customize security descriptors
```

Comment:
Do we need to manage what version this section shows up in?

Reply:
Yes, I'll make a .NET 10 version